lms/envs/aws.py

```diff
@@ -119,8 +121,8 @@
     """
 
     'collect_stats': {
-    'task': 'openedx.core.djangoapps.edx_global_analytics.tasks.collect_stats',
-    'schedule': 30, # number in seconds.
+        'task': 'openedx.core.djangoapps.edx_global_analytics.tasks.collect_stats',
+        'schedule': crontab(hour=0, minute=random.randint(1, 59))
```

Comment:
Configurability is a good point.
The thing is admins usually edit `.json` config files. Not this `aws.py` file.

Also currently you're introducing `celery` dependency into this file. I would rather not do that.

On the other hand, it looks like `periodic_task` decorator has been removed from the celery.
Ok, let's leave this as is for now.